.github/workflows/update-holonix-version.yml

```diff
@@ -19,27 +19,39 @@ concurrency:
 
 jobs:
   update-holonix-version:
-    runs-on: [ self-hosted, multi-arch ]
+    runs-on: "ubuntu-latest"
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
+      - uses: actions/checkout@v4
+        with:
+          ref: develop
+      - uses: cachix/install-nix-action@v22
+      - name: Get tags
+        run: git fetch --quiet --tags origin
+      - name: Configure Git
+        shell: bash
+        run: |
+          ./scripts/ci-git-config.sh
       - name: Run the update script
         run: |
           ./scripts/update-holonix-version.sh ${{ github.event.inputs.version }}
```

Comment:
What does `-E` do? I can't find that in the man page for `set`.

Both scripts set these flags and `-e` must be set in the context that actions run the scripts in because it must stop on error to be a correct CI system.

Reply:
i think at least having `-e` is critical to ensure here.